content/migrate/tf2pulumi.md

```diff
@@ -79,7 +79,7 @@ form:
 </div>
 
 <div id="pulumi-code-download-icon" class="float-right mt-4 mr-1 hidden">
-    <button class="copy-button" onclick="downloadCode()"><i class="fa fa-download text-xl" title="Download"></i></button>
+    <button class="copy-button" onclick="downloadCode('tf2pulumi.zip')"><i class="fa fa-download text-xl" title="Download"></i></button>
```

Comment:
Download link seemed to generate a zip called kube2pulumi.zip on all pages.

Reply:
Nice catch.